llm_bench/python/who_what_benchmark/requirements.txt

```diff
@@ -3,6 +3,8 @@ sentence-transformers>=2.2.2
 openvino>=2023.3.0
 openvino-telemetry>=2023.2.1
 optimum-intel>=1.14
+openvino-tokenizers>=2023.3.0
+openvino-genai>=2024.3.0
```

Comment:
I suppose it will not work if openvino, openvino-tokenizers and openvino-genai installed with different version. Possibly it is better to set the same lower bound for all 3 packages